lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -104,6 +105,21 @@ def server_version
       kubectl.server_version
     end
 
+    # Initializes the task
+    #
+    # @param namespace [String] Kubernetes namespace
+    # @param context [String] Kubernetes context / cluster
```

Comment:
```suggestion
    # @param context [String] Kubernetes context
```

Reply:
Context = cluster + user we use them interchangeably since we always auth the same way, but they technically aren't the same.